src/forms/services/form-submission.service.ts

```
@@ -0,0 +1,1184 @@
+          queryBuilder.andWhere(
+            'fv.fieldId = :fieldId AND (fv.textValue ILIKE :textSearchValue OR fv.value ILIKE :textSearchValue)',
+            {
+              fieldId,
+              textSearchValue: `%${textSearchValue}%`
+            }
+          );
+          break;
+
+        case FieldType.TEXTAREA:
+          // Check textareaValue column first, then fall back to value column with ILIKE
+          const textareaSearchValue = expectedValue.toString();
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)